internal/acctest/vcr.go

```diff
@@ -276,12 +273,12 @@ func vcrProviderConfigureContextFunc(provider *schema.Provider, configureContext
 		// TODO Need to loop through all API clients to do this.
 		// TODO Use []*client.Client?
 		// TODO AWS SDK for Go v2 API clients.
-		meta.LogsConn(ctx).Handlers.AfterRetry.PushFront(func(r *request.Request) {
-			// We have to use 'Contains' rather than 'errors.Is' because 'awserr.Error' doesn't implement 'Unwrap'.
-			if errs.Contains(r.Error, cassette.ErrInteractionNotFound.Error()) {
-				r.Retryable = aws.Bool(false)
-			}
-		})
+		// meta.LogsConn(ctx).Handlers.AfterRetry.PushFront(func(r *request.Request) {
+		// 	// We have to use 'Contains' rather than 'errors.Is' because 'awserr.Error' doesn't implement 'Unwrap'.
+		// 	if errs.Contains(r.Error, cassette.ErrInteractionNotFound.Error()) {
+		// 		r.Retryable = aws.Bool(false)
+		// 	}
+		// })
```

Comment:
Can this be permanently removed now?

Reply:
I left it there as a reminder of what we want to do for VCR -- now that we have the API clients lazy loaded I _think_ we can eventually add this to the generated code.

Reply:
Makes sense, thanks!